src/main/drivers/vtx_soft_spi_rtc6705.c

```diff
@@ -123,7 +123,21 @@ void rtc6705_soft_spi_set_channel(uint16_t channel_freq)
 
 void rtc6705_soft_spi_set_rf_power(uint8_t reduce_power)
 {
-    rtc6705_write_register(7, (reduce_power ? (PA_CONTROL_DEFAULT | PD_Q5G_MASK) & (~(PA5G_PW_MASK | PA5G_BS_MASK)) : PA_CONTROL_DEFAULT));
+    switch(reduce_power)
+    {
+        case 0:// 1dBm
+            rtc6705_write_register(7, 0x006A);
```

Comment:
Have you tested these with a rf power meter? 

My tests with a 25mW tx5813 (spi mod) show this to increase the power to maximum. 
Even by viewing the current draw from my power supply shows a significant increase in current draw.

I created a cli to test the rtc6705 chip and experiment with the registers and all of these values listed here have the opposite effect on the power output.

Below are my observations. Tested at 5800MHz using Immersion RF Power Meter with the vtx at operating temperature.
```
Hex             Binary          Power  comment
0x0006A  B0000000000001101010   64mW   Maximum power
0x04C7D  B0000100110001111101   48mW   
0x04D0D  B0000100110100001101   50mW
   -              -               -        -
0x04FBD  B0000100111110111101   34mW   default value
0x24FBD  B0100100111110111101   26mW
0x44FBD  B1000100111110111101   13mW
0x64FBD  B1100100111110111101    7mW
0x7CFBD  B1111100111110111101    4mW
0x78FC0  B1111100111111000000    1mW   min power
```
Notice after the break I'm only changing bits 16-19 and getting much better control of the output power. (increasing the last 4 bits decreases the output power.)

Also note that register 7 is reset any time register 1 is changed. 
**This means that you will have to reapply the power setting every time you change frequencies to maintain the desired power level.**